src/mixins/itext_click_behavior.mixin.js

```diff
@@ -152,7 +152,7 @@ fabric.util.object.extend(fabric.IText.prototype, /** @lends fabric.IText.protot
    */
   mouseUpHandler: function(options) {
     this.__isMousedown = false;
-    if (!this.editable || this.group ||
+    if (!this.editable ||
```

Comment:
should we keep here  `|| ( this.group && !this.group.interactive )` ?

Reply:
af3430ce

Reply:
af3430ce